src/hoc/Aux/Aux.js

```diff
@@ -0,0 +1,3 @@
+const aux = (props) => props.children;
```

Comment:
Remove `Aux.js` you can use `<>` `</>` as aux as we discussed.

Reply:
Need to fix this.